third_party/java/java_tools/README.md

```diff
@@ -14,3 +14,35 @@ third_party/java/java_tools/turbine_direct_binary_deploy.jar
 third_party/java/java_tools/JavaBuilder_deploy.jar
 third_party/java/java_tools/jarjar_command_deploy.jar
 third_party/java/java_tools/Runner_deploy.jar
+
+The following tools were built with bazel 0.22.0 at commit 569e2cb4d1b7653b5a02bbf664765476233e6c33
```

Comment:
Why twice?

Reply:
You may want to replace the file and not append it. You have the history in version control.

Reply:
/cc @iirina -- this was generated by your script.

Done.